PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
+
+`docker run -d --name s3server -p 8000:8000 -e HOST_NAME=new.host.com
+scality/s3server`
+
+### ACCESS_KEY and SECRET_KEY
+
+These variables specify a Scality S3server user credentials named Docker:
+
+`docker run -d --name s3server -p 8000:8000 -e ACCESS_KEY=newAccessKey -e
+SECRET_KEY=newSecretKey scality/s3server`
+
+### LOG_LEVEL
+
+This variable allows you to change the log level: info, debug or trace:
+
+`docker run -d --name s3server -p 8000:8000 -e LOG_LEVEL=trace scality/s3server`
+
+
+# Using Scality S3server in production with Docker
+
+## Using Docker Volume in production
+
+S3Server runs with a file backend by default.
+
+So, by default, the data are stored inside your S3server Docker container.
+
+However, you **MUST** use Docker volumes to host the data and the metadata of
+the S3server container outside your S3server Docker container.
+Otherwise, data are destroyed after erasing the container.
+
+`docker run -­v $(pwd)/data:/usr/src/app/localData -­v
+$(pwd)/metadata:/usr/src/app/localMetadata -p 8000:8000 ­-d scality/s3server`
+
+## Adding/ modifying/ deleting accounts/ users credentials
+
+1. Create locally a customized `conf/authdata.json`.
+
+2. Use [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/)
+
+to override the default one through a docker file mapping.
+
+`docker run -v $(pwd)/authdata.json:/usr/src/app/conf/authdata.json -p`
+`8000:8000 -d scality/s3server`
+
+## Setting a new region
+
+To specify an host name (e.g. s3.domain.name),
+you can provide your own
+[config.json](https://github.com/scality/S3/blob/master/config.json)
+using [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/).
+
+First, add a new region to your config.json that you created locally:
```

Comment:
First add a new key-value pair in the regions section of your config.json.  The key in the key-value pair should be your "region" name and the value is an array containing any host name you would like to add.